templates/organization/planning/_results.html.twig

```diff
@@ -54,7 +54,7 @@
 {% set availabilities = getAvailabilities(periodCalculator, filters) %}
 <div class="planning-container">
     {% if availabilities.assets|length > 0 or availabilities.users|length > 0 %}
-        <table class="planning table table-sm table-bordered availability-table" data-availability-mode="planning" style="display: none;">
+        <table class="planning table table-sm table-bordered availability-table" data-availability-mode="planning" {#style="display: none;"#}>
```

Comment:
this is temporary and will be reverted once #344  will be merged

Reply:
We cannot merge this PR as it. This line must be reverted before, as it's critical for performances

Reply:
We should add a comment here in order to explain why